lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -31,6 +31,9 @@ def air_loop_hvac_apply_prm_baseline_fan_power(air_loop_hvac)
     # Get system type associated with air loop
     system_type = air_loop_hvac.additionalProperties.getFeatureAsString('baseline_system_type').get
 
+    # Get the fan limitation pressure drop adjustment bhp
+    fan_pwr_adjustment_bhp = air_loop_hvac_fan_power_limitation_pressure_drop_adjustment_brake_horsepower(air_loop_hvac)
+
```

Comment:
Add the pressure drop adjustment function, right now, it pretty much always returns 0 but I know that we're going to want to update it based on user inputs.